.github/workflows/gh-pages.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v2
         with:
-          node-version: '14'
+          node-version: '15'
```

Comment:
Why not 16?

Reply:
I was having trouble building locally, but now everything seems to be working so we'll give 16 a try.